### PR TITLE
PLT-6955: Add new config fields to diagnostics.

### PR DESCRIPTION
**Pull request by grundleborg**
#### Summary
Add new config fields to diagnostics.

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-6955

### CODE REVIEW

File: app/diagnostics.go [hunk 1]
```diff
@@ -241,34 +247,37 @@ func trackConfig() {
...
-		"thumbnail_height":    utils.Cfg.FileSettings.ThumbnailHeight,
-		"preview_width":       utils.Cfg.FileSettings.PreviewWidth,
-		"preview_height":      utils.Cfg.FileSettings.PreviewHeight,
-		"profile_width":       utils.Cfg.FileSettings.ProfileWidth,
-		"profile_height":      utils.Cfg.FileSettings.ProfileHeight,
-		"max_file_size":       *utils.Cfg.FileSettings.MaxFileSize,
+		"enable_public_links":     utils.Cfg.FileSettings.EnablePublicLink,
+		"driver_name":             utils.Cfg.FileSettings.DriverName,
+		"amazon_s3_ssl":           *utils.Cfg.FileSettings.AmazonS3SSL,
+		"amazon_s3_signv2":        *utils.Cfg.FileSettings.AmazonS3SignV2,
```
Comment (jasonblais) [inline app/diagnostics.go]: I think this should be `amazon_s3_sign_v2`

File: app/diagnostics.go [hunk 2]
```diff
@@ -366,6 +378,13 @@ func trackConfig() {
 	SendDiagnostic(TRACK_CONFIG_ANALYTICS, map[string]interface{}{
 		"isdefault_max_users_for_statistics": isDefault(*utils.Cfg.AnalyticsSettings.MaxUsersForStatistics, model.ANALYTICS_SETTINGS_DEFAULT_MAX_USERS_FOR_STATISTICS),
 	})
+
+	SendDiagnostic(TRACK_CONFIG_ANNOUNCEMENT, map[string]interface{}{
+		"enable_banner":          *utils.Cfg.AnnouncementSettings.EnableBanner,
+		"banner_color":           isDefault(*utils.Cfg.AnnouncementSettings.BannerColor, model.ANNOUNCEMENT_SETTINGS_DEFAULT_BANNER_COLOR),
```
Comment (jasonblais) [inline app/diagnostics.go]: Can we call this `isdefault_banner_color` so it's consistent with other settings

File: app/diagnostics.go [hunk 3]
```diff
@@ -366,6 +378,13 @@ func trackConfig() {
...
+		"banner_text_color":      isDefault(*utils.Cfg.AnnouncementSettings.BannerTextColor, model.ANNOUNCEMENT_SETTINGS_DEFAULT_BANNER_TEXT_COLOR),
```
Comment (jasonblais) [inline app/diagnostics.go]: And this `isdefault_banner_text_color`

File: app/diagnostics.go (hunk 1 above)
Comment (grundleborg) [inline app/diagnostics.go]: Discussed with @jasonblais on pre-release. Kept the typo version as we were already tracking with the typo and we don't want a change of key.

**Pull request merged**